manuscripts/esquery.py

```diff
@@ -64,7 +69,7 @@ def __get_query_filters(cls, filters={}, inverse=False):
             # trying to use es_dsl only and not creating hard coded queries
             query_filters.append(Q('match_phrase', **params))
 
-        return query_filters
+        return sorted(query_filters, key=lambda x: sort_order(x))
```

Comment:
I think it is better to do this in tests, instead of here. The main reason is performance (this is not really needed when working as usual, but the performance penalty of ordering will be paid every time you query), but also that the code does not ready this added complexity: tests do need, but not this code....